vault/dynamic_system_view.go

```diff
@@ -23,8 +23,9 @@ func (c ctxKeyForwardedRequestMountAccessor) String() string {
 }
 
 type dynamicSystemView struct {
-	core       *Core
-	mountEntry *MountEntry
+	core        *Core
+	mountEntry  *MountEntry
+	perfStandby bool
```

Comment:
Do we need to add this new struct member to testing.go? https://github.com/hashicorp/vault/pull/17186/files#file-vault-testing-go-L511

Reply:
Yes thanks.